src/fheroes2/gui/player_info.cpp

```diff
@@ -427,9 +427,9 @@ bool Interface::PlayersInfo::QueueEventProcessing()
         if ( player != nullptr ) {
             if ( conf.getCurrentMapInfo().AllowChangeRace( player->GetColor() ) ) {
```

Comment:
To make Sonar a little happier we should combine these two conditions:
```suggestion
        if ( player != nullptr && conf.getCurrentMapInfo().AllowChangeRace( player->GetColor() ) ) {
```